benchexec/tools/depthk.py

```diff
@@ -36,17 +40,16 @@ class Tool(benchexec.tools.template.BaseTool):
 
     def executable(self):
 
-        return Util.find_executable('depthk-wrapper.sh')
+        return Util.find_executable('../../svcomp/depthk/depthk-wrapper.sh'
+                                    )
 
     def working_directory(self, executable):
         executableDir = os.path.dirname(executable)
         return executableDir
 
     def version(self, executable):
-        version = subprocess.Popen([executable, '-v'],
-                                   stdout=subprocess.PIPE,
-                                   stderr=subprocess.STDOUT).stdout.readline().decode()
-        return version
+
+        return 'DepthK 3.0'
```

Comment:
As I have repeatedly said, please do not hard-code version numbers in the tool-info!